client/scraper/wikipedia_metadata.py

```diff
@@ -1,8 +1,10 @@
+from bs4 import BeautifulSoup
```

Comment:
can you split the imports like
third party imports

python library imports

in app imports